### PR TITLE
Clear spike vectors after writing or transferring.

### PR DESCRIPTION
**Pull request by nrnhines**
Fixes Ivan's last problem mentioned in #369.

### CODE REVIEW

**Comment by nrnhines** [comment]
Oops. I inadvertently committed as well my private change to the spike output format.

**Comment by nrnhines** [comment]
The errors on the default pipeline are
```
[ERROR] No output files. Test failed!
```
Not clear to me why this change caused that.

**Comment by pramodk** [comment]
> Not clear to me why this change caused that.

I believe `{ }` were missing to the if statement. Fixes that now.

File: coreneuron/io/output_spikes.cpp [hunk 1]
```diff
@@ -268,8 +268,10 @@ void output_spikes_serial(const char* outpath) {
 
 void output_spikes(const char* outpath, const std::string& population_name) {
     // try to transfer spikes to NEURON. If successfull, don't write out.dat
-    if (all_spikes_return(spikevec_time, spikevec_gid))
+    if (all_spikes_return(spikevec_time, spikevec_gid)) {
+        clear_spike_vectors();
         return;
+    }
```
Comment (nrnhines) [inline coreneuron/io/output_spikes.cpp:274]: Never fails to surprise me that something can be right in front of my nose and I can't see it til someone says, "see!"

**Pull request merged**